src/components/sidenav/sidenav.ts

```diff
@@ -286,9 +281,6 @@ export class MdSidenavLayout implements AfterContentInit {
    */
   private _validateDrawers() {
     this._start = this._end = null;
-    if (this._sidenavs.length === 0) {
-      throw new MdMissingSidenavException();
-    }
```

Comment:
Add a test? While the code isn't explicitly throwing an exception when a sidenav is missing, having a test would guard against accidentally throwing an error if the sidenav is missing (such as, for example, a null dereference).